### ISSUE TITLE
#2228 Need to find a suitable LR

### ISSUE THREAD

**Issue by Ishank2301** [Ishank2301/Pratise-ML-Algorithums#2]
Through the initial assumptions and deductions it feels like we need to find a good Learning Rate as the model is undertrained

**Comment by Ishank2301** [comment]
Used the serial format #227 ffrom the keras documendation used kerastuner to resolve the issue by setting Dynamic Adaption of various Hyperparameters, Number of Hidden layers and Each layer containing specific amount of neuron, the Dead neuron cycle was efficiently resolved using  Choice function of the tuner metaphase function and the Dropout layer + Dynamic selection of Activation functions and Optimizers.

In future i might try to fit in the Regularizers to  gain a weighted Biases and resolve more issues to make our model more Efficient need to determine a way a select efficiently which optimizers to use and what specific values should be use in Edge and Catastrophical cases.
